pkg/tracingpolicy/generictracingpolicy_test.go

```diff
@@ -527,3 +528,78 @@ func TestYamlNamespaced(t *testing.T) {
 	_, ok := tp.(TracingPolicyNamespaced)
 	require.True(t, ok)
 }
+
+func createTempFile(t *testing.T, data string) string {
+	file, err := os.CreateTemp(t.TempDir(), "tetragon-")
+	if err != nil {
+		t.Error(err)
```

Comment:
I think you should use `Fatal` here since you cannot recover from this and you should stop the test here.